src/ALE/MOM_regridding.F90

```diff
@@ -1371,6 +1376,8 @@ subroutine build_rho_grid( G, GV, US, h, tv, dzInterface, remapCS, CS )
   endif
 
   nz = GV%ke
+  ice_shelf = .false.
+  if (present(frac_shelf_h)) ice_shelf = .true.
```

Comment:
These 2 lines could be written more simply as `ice_shelf = present(frac_shelf_h)`.